docs/Developer.md

```diff
@@ -74,9 +74,9 @@ Workflow details (getting the source code in blue, pull request in red, integrat
 
 ### Refactoring contributions from now on, and scientific contributions after cycle 49t1
 
-Who: developers with scientific contributions based on cycles 49t2 and following; and developers with GPU-refactoring contributions.
+Who: developers with scientific contributions based on cycles 49t1 and following; and developers with GPU-refactoring contributions.
```

Comment:
Work based on 49t1 follows the previous workflow. This chapter concerns contributions based on 49t2 and followings.

Reply:
Same, there will not be contributions based on 49t2. If that cycle has to exist, the contributions to build it will be based on 49t1.